README.md

```diff
@@ -101,8 +101,8 @@ In addition to using the docker image, you may run the pipeline directly from
 source. First install git, python, pip, and virtualenv:
 
 ```bash
-sudo apt-get install -y git python-pip python-dev build-essential
-sudo pip install --upgrade pip
+sudo apt-get install -y git python-pip python-dev build-essential  &&
+sudo pip install --upgrade pip &&
```

Comment:
Although the `&&` works, I think its fine to also enter line by line. I'd rather not include it in the docs.